subsys/net/ip/dhcpv4.c

```diff
@@ -117,6 +117,12 @@ static bool dhcpv4_add_req_ipaddr(struct net_pkt *pkt,
 					      4, addr->s4_addr);
 }
 
+static bool dhcpv4_add_hostname(struct net_pkt *pkt, const char *hostname)
```

Comment:
Don't we need to add `#if defined(CONFIG_NET_HOSTNAME_ENABLE)` here too?

Reply:
My bad, thank you.